nupic/algorithms/anomaly.py

```diff
@@ -157,6 +143,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
+# implementations of compute() function
+  def computeRaw(self, active, prevPredicted, inputValue=None, timestamp=None):
+    return computeRawAnomalyScore(active, prevPredicted)
+
+  def computeLikelihood(self, active, prevPredicted, inputValue, ts=None):
+    """Anomaly computed using the anomaly_likelihood score, 
+       which models probability of (input, anomalyScore) pair.
+    """
+    assert isinstance(self._likelihood, AnomalyLikelihood)
+    if inputValue is None:
+      raise ValueError("Selected anomaly mode 'Anomaly.MODE_LIKELIHOOD' "
+                       "requires 'inputValue' as parameter to compute() method. ")
+    rawScore = computeRawAnomalyScore(active, prevPredicted)
+    probability = self._likelihood.anomalyProbability(inputValue, rawScore, ts)
+    return probability
```

Comment:
I didn't understand why this is returning the probability and not the score (`score = 1 - probability`) as was done earlier (https://github.com/numenta/nupic/pull/1934/files#diff-ac4ff5014c0e81ed147be47c27c7bc7aL140). Are you sure that this won't cause problems in code that uses this function?